ui/submodules/packages/components/src/components/usagesTable.tsx

```diff
@@ -24,45 +29,67 @@ export function UsagesTable(props: TableViewData<ComponentUsageType>): JSX.Eleme
                 field: "name",
                 cellClassName: "noPadding stretch",
                 headerName: t("table.usages.title.NAME", "Name"),
-                flex: 2,
+                flex: 3,
+                minWidth: 160,
                 renderCell: ScenarioCell,
+                hideable: false,
             },
             {
-                field: "isSubprocess",
-                headerName: t("table.usages.title.IS_FRAGMENT", "Fragment?"),
+                field: "isFragment",
+                headerName: t("table.usages.title.IS_FRAGMENT", "Fragment"),
+                valueGetter: ({ row }) => row.isSubprocess,
                 type: "boolean",
             },
             {
                 field: "processCategory",
-                headerName: t("table.usages.title.PROCESS_CATEGORY", "Process category"),
+                headerName: t("table.usages.title.PROCESS_CATEGORY", "Category"),
                 renderCell: Highlighted,
-            },
-            {
-                field: "modificationDate",
-                headerName: t("table.usages.title.MODIFICATION_DATE", "Modification date"),
-                type: "dateTime",
                 flex: 1,
-                renderCell: Highlighted,
             },
             {
                 field: "createdAt",
                 headerName: t("table.usages.title.CREATION_DATE", "Creation date"),
                 type: "dateTime",
-                flex: 1,
+                flex: 2,
                 renderCell: Highlighted,
+                hide: true,
             },
             {
                 field: "createdBy",
-                headerName: t("table.usages.title.CREATED_BY", "Created by"),
+                headerName: t("table.usages.title.CREATED_BY", "Author"),
                 renderCell: Highlighted,
+                flex: 1,
+            },
+            {
+                field: "modificationDate",
+                headerName: t("table.usages.title.MODIFICATION_DATE", "Modification date"),
+                type: "dateTime",
+                flex: 2,
+                renderCell: Highlighted,
+            },
+            {
+                field: "isDeployed",
+                headerName: t("table.usages.title.IS_DEPLOYED", "Deployed"),
+                valueGetter: ({ row }) => isDeployed(row),
+                type: "boolean",
+                renderCell: ({ value }) => {
+                    if (value === null) {
```

Comment:
Why not:

Last action null: https://staging.nussknacker.io/static/assets/states/not-deployed.svg
Deployed: https://staging.nussknacker.io/static/assets/states/deploy-success.svg
Canceled: https://staging.nussknacker.io/static/assets/states/stopping-success.svg

?